src/start/HelpScreen.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { Text, StyleSheet } from 'react-native';
+
+import { Screen, RawLabel, Centerer } from '../common';
+
+export default class HelpScreen extends PureComponent<{}> {
```

Comment:
How about `WelcomeHelpScreen`? There could be other help screens, and this sounds like something generic for all of them.

Reply:
That makes sense, will change.